deploy/crds/toolchain.openshift.dev_cheinstallations.yaml

```diff
@@ -1,6 +1,9 @@
+
```

Comment:
I need to check why we have this empty line and separator at the beginning of this file...

Reply:
It's because controller-gen generates multiple manifest file which cause issues. See https://github.com/codeready-toolchain/api/blob/master/make/generate.mk#L78-L79
We should either remove it or switch to operator-sdk generator which doesn't have that issue.

Another problem is with `type: object` in `validation.openAPIV3Schema.properties` path which is not supported by Kube 1.11 (== minishift). But I think operator-sdk will generate it too.
So, we should either remove it from the generated CRD as we do it in api repo or drop minishift support completely. I'm fine with both options.

Reply:
I'm fine with dropping support for Minishift at this point. 
@tinakurian @MatousJobanek @sbryzak @nurali-techie @michaelkleinhenz ok for you as well?